util/test/chpl_launchcmd.py

```diff
@@ -568,7 +568,10 @@ class OurFormatter(argparse.ArgumentDefaultsHelpFormatter, argparse.RawDescripti
 
         # Allow hostlist to be set in environment variable CHPL_LAUNCHCMD_HOSTLIST.
         if args.hostlist is None:
-            args.hostlist = os.environ.get('CHPL_LAUNCHCMD_HOSTLIST')
+            env_hostlist = os.environ.get('CHPL_LAUNCHCMD_HOSTLIST')
+            if not env_hostlist:
+                env_hostlist = None
+            args.hostlist = env_hostlist
 
```

Comment:
Might be a little cleaner written as: 

if args.hostlist is None:
    env_hostlist = os.environ.get('CHPL_LAUNCHCMD_HOSTLIST')
    if env_hostlist:
        args.hostlist = env_hostlist


or maybe: 
if args.hostlist is None:
    args.hostlist = os.environ.get('CHPL_LAUNCHCMD_HOSTLIST') or None

?